lib/sleet/cli.rb

```diff
@@ -20,6 +20,15 @@ class Cli < Thor
     option :output_file, type: :string, aliases: [:o], desc: <<~DESC
       This is the name for the output file, on your local system. It is relative to the source_dir. Will be IGNORED if workflows is provided.
     DESC
+    option :username, type: :string, aliases: [:u], desc: <<~DESC
+      This is the GitHub username that is referenced by the CircleCI build. By default, Sleet will base this on your upstream git remote.
+    DESC
+    option :project, type: :string, aliases: [:p], desc: <<~DESC
```

Comment:
I excluded `github_` from these arg names because I feel like it's really only 1 step away from supporting other remote services. Like, if you specify "gitlab" as the remote, you just need to _also_ specify username, project & branch, and the CircleCI lookups should just work.

But I left it out of this PR because I didn't want to get too far ahead of myself.

Reply:
Ya I think you're right that Gitlab support is well within reach here! Totally makes sense to leave it out for now/split it up!

I'll create another issue here in a sec to track that!